### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by arunkg2787**
Fixes #https://github.com/grafana/support-escalations/issues/5769

Image Rendering : Added more context wrt grafana cloud

**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by GrafanaWriter** [comment]
Please follow the Writers' Toolkit with respect to using [numbered steps for tasks](https://grafana.com/docs/writers-toolkit/structure/topic-types/task/). We will make comments in this PR, but please use this information for future PRs.

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 1]
```diff
@@ -40,6 +44,15 @@ Starting from Grafana v7.0.0, all PhantomJS support has been removed. Please use
 
 To install the plugin, refer to the [Grafana Image Renderer Installation instructions](/grafana/plugins/grafana-image-renderer/?tab=installation#installation).
 
+## Testing Image Rendering
+Once you have enabled image rendering, you can test if it is working by following these steps:
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:48]:
```suggestion
Once you have enabled image rendering, complete the following steps to test it is working:
```

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 2]
```diff
@@ -40,6 +44,15 @@ Starting from Grafana v7.0.0, all PhantomJS support has been removed. Please use
...
+
+* Go to a panel that you want to render as an image.
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:50]:
```suggestion
1. Navigate to a panel that you want to render as an image.
```

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 3]
```diff
@@ -40,6 +44,15 @@ Starting from Grafana v7.0.0, all PhantomJS support has been removed. Please use
...
+* Click the dropdown arrow next to the panel title.
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:51]:
```suggestion
1. Click the dropdown arrow next to the panel title.
```

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 4]
```diff
@@ -40,6 +44,15 @@ Starting from Grafana v7.0.0, all PhantomJS support has been removed. Please use
...
+* Click Share > Direct link rendered image.
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:52]:
```suggestion
1. Click **Share > Direct link rendered image**.
```

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 5]
```diff
@@ -40,6 +44,15 @@ Starting from Grafana v7.0.0, all PhantomJS support has been removed. Please use
...
+
+If image rendering is working correctly, you will see a link to the rendered image. You can click on the link to open the image in a new tab.
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:54]:
```suggestion
If image rendering works correctly, you will see a link to the rendered image. You can click the link to open the image in a new tab.
```

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 6]
```diff
@@ -466,3 +479,9 @@ RENDERING_VIEWPORT_PAGE_ZOOM_LEVEL=1
   }
 }
 ```
+## Features Not Available in Grafana Cloud
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:482]:
```suggestion
## Features not available in Grafana Cloud
```

**Review by chri2547 — COMMENTED** [review]
@arunkg2787 , thank you for the contribution! I made some suggestions to format the content. Looks good, please accept suggestions and ping me for approval and merge.

You will also need to run prettier. Please let me know if you don't know how to do that and I can take care of it.

**Comment by irenerl24** [comment]
fixed in https://github.com/grafana/grafana/pull/95571

**Pull request closed without merging**